Dockerfile

```diff
@@ -20,21 +20,21 @@ RUN /etc/init.d/postgresql start &&\
 
 #build osm2pgsql
 USER root
-RUN git clone git://github.com/openstreetmap/osm2pgsql.git ~postgres/src/osm2pgsql --depth 1
+RUN git clone https://github.com/openstreetmap/osm2pgsql.git ~postgres/src/osm2pgsql --depth 1
 RUN apt-get -y install make cmake g++ libboost-dev libboost-system-dev libboost-filesystem-dev libexpat1-dev\
-  zlib1g-dev libbz2-dev libpq-dev libgeos-dev libgeos++-dev libproj-dev lua5.2 liblua5.2-dev osmium-tool
+  zlib1g-dev libbz2-dev libpq-dev libgeos-dev libgeos++-dev libproj-dev lua5.2 liblua5.2-dev osmctools
 RUN cd ~postgres/src/osm2pgsql && mkdir build && cd build && cmake .. && make && make install
 
 #install Mapnik
 RUN apt-get -y install autoconf apache2-dev libtool libxml2-dev libbz2-dev libgeos-dev libgeos++-dev\
   libproj-dev gdal-bin libmapnik-dev mapnik-utils python-mapnik sudo
 
 #build mod_tile and renderd
-RUN git clone https://github.com/openstreetmap/mod_tile.git ~postgres/src/mod_tile --depth 1
-RUN cd ~postgres/src/mod_tile && ./autogen.sh && ./configure && make && make install && make install-mod_tile && ldconfig
+RUN git clone https://github.com/openstreetmap/mod_tile.git ~postgres/src/mod_tile
+RUN cd ~postgres/src/mod_tile && git reset fd5988fc5877c51838ad96991d6e2912cfaf7d61 --hard && ./autogen.sh && ./configure && make && make install && make install-mod_tile && ldconfig

```

Comment:
Can you add a comment explaning why this specific commit is necessary? (And maybe a TODO to iron out the issue requiring it to be pinned?)

Reply:
@thecristen do you happen to remember the answer to this? This was the one part of your changes that I didn't really understand. 😅

Reply:
I no longer remember! Apologies @lemald, maybe you can revert these two lines and report back if it produces an error? :P

Reply:
Okay, it looks like there are a lot of other changes we would need to make with the packages that are being installed to make things work on 22.04, not to mention potential complications with newer versions of some packages. @ianwestcott would you be all right if I added a note explaining what needs to be done in the future rather than trying to get it all working as part of this PR?

Reply:
That's a perfect amount of context to facilitate future improvements, thanks @lemald!